assets/scss/main.scss

```diff
@@ -14,3 +14,9 @@
 //
 @import "accordion";
 @import "popover";
+
+//
+// SWARM-ANIMATION
+//
+@import "~swarm-animation/src/constants";
+@import "~swarm-animation/src/fade";
```

Comment:
For reference, consumers of `meetup-web-components` import `assets/scss/components/_all.scss`. If an individual component needs something from `swarm-animation`, it should be imported in `_all.scss` or in the component partial